linux_os/guide/system/accounts/accounts-banners/banner_etc_issue/ansible/shared.yml

```diff
@@ -5,7 +5,26 @@
 # disruption = medium
 {{{ ansible_instantiate_variables("login_banner_text") }}}
 
-- name: "{{{ rule_title }}} - ensure correct banner"
+{{% if product not in ['sle15'] %}}
```

Comment:
You can remove some empty lines left by Jinja2 in the resulting file:
https://jinja.palletsprojects.com/en/3.0.x/templates/#whitespace-control

Also valid for other files in this PR.

Reply:
Sorry for some reason missed the message long time ago and the PR was displayed in the list as review required, so I thought you don't have time for it :)

Anyways modified those macros fromm `{{% %}}` to `{{%- -%}}`, but am a bit confused since I can see extra new lines , after build, on my local setup only in the bash files. Could it be because I am using ninja build or something else. I could not find anything in my environment that should cause the ssg build procedure to consider trimming default behaviour.